stl/inc/format

```diff
@@ -2859,122 +2862,142 @@ _NODISCARD auto make_wformat_args(const _Args&... _Vals) {
     return _Format_arg_store<wformat_context, _Args...>{_Vals...};
 }
 
-template <class _Out, class _CharT>
-using format_args_t = basic_format_args<basic_format_context<_Out, _CharT>>;
+template <class _CharT, class... _Args>
+struct _Fmt_basic_string {
+    basic_string_view<_CharT> _Str;
+
+    template <class _Ty, enable_if_t<convertible_to<const _Ty&, basic_string_view<_CharT>>, int> = 0>
+    constexpr _Fmt_basic_string(const _Ty& _Str_) : _Str{_Str_} {}

```

Comment:
There should be a transition comment regarding the msvc bug.

Reply:
Added